### PR TITLE
Gp2 680 restyle search modal

### PR DESCRIPTION
**Pull request by bobmeredith**
## To do (delete all that do not apply):

 - [x] Ticket exists in Jira  https://uktrade.atlassian.net/browse/GP2-680 
 - [x] Jira ticket has the correct status.
 - [x] [Changelog](CHANGELOG.md) entry added.

 - [x] (if is a UI change) Includes screenshot(s) - ideally before and after, but at least after
 - [x] (if needs manual testing) How to test locally, including how to set up appropriate data 
 - [X] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Comment by bobmeredith** [comment]
image

**Comment by bobmeredith** [comment]
Testing - This is mainly a restyle however:
following from ACs, the search button should be disabled until chararcters are entered .
On clicking the clear cross - the search input box should clear, the search button disable and the search box remain focussed.

File: react-components/src/components/ProductFinder/ProductFinder.jsx [hunk 1]
```diff
@@ -284,24 +282,36 @@ export function ProductFinder(props) {
...
+      <ReactModal 
+        isOpen={modalIsOpen} 
+        onRequestClose={closeModal} 
+        className="modal max-modal p-v-s p-h-l"
+        overlayClassName="modal-overlay center"
+        onAfterOpen={modalAfterOpen}
+      >
         <form className="product-finder">
-          <div className="search-header bg-blue-deep-80 text-white p-s" style={{ height: '172px' }}>
+          <div className="modal-header" style={{ height: '172px' }}>
```
Comment (lewis-coulson-dit) [inline react-components/src/components/ProductFinder/ProductFinder.jsx:295]: No biggie but could be helpful to put that in a var

File: react-components/src/components/ProductFinder/ProductFinder.jsx [hunk 2]
```diff
@@ -284,24 +282,36 @@ export function ProductFinder(props) {
       {/* eslint-disable-next-line react/button-has-type */}
       <button className={buttonClass} onClick={openModal}>
         {selectedProduct || 'add product'}
-        <i className="fas fa-chevron-right"></i>
+        <i className={'fa ' + (selectedProduct ? 'fa-edit' : 'fa-plus')}></i>
```
Comment (dahfool) [inline react-components/src/components/ProductFinder/ProductFinder.jsx:285]: This is invalid JSX should be `<i className='fa fa-plus' />`

Comment (dahfool) [reply react-components/src/components/ProductFinder/ProductFinder.jsx:285]: Might need updating in other places.

File: react-components/src/components/ProductFinder/ProductFinder.jsx (hunk 1 above)
Comment (bobmeredith) [reply react-components/src/components/ProductFinder/ProductFinder.jsx:295]: Yup - This is the first in a bunch of changes - and I just did that exact thing for the second bit.

File: react-components/src/components/ProductFinder/ProductFinder.jsx [hunk 3]
```diff
@@ -6,22 +6,7 @@ import Services from '@src/Services'
...
-    padding: '0',
-    border: 'none',
-    overflow: 'none'
-  },
-  overlay: {
-    background: 'rgb(45 45 45 / 45%)',
-    zIndex: '3'
-  }
-}
+const customStyles = {}
```
[the remark this answers is not in the document]
Comment (bobmeredith) [reply react-components/src/components/ProductFinder/ProductFinder.jsx:9]: Good point - will clean

File: react-components/src/components/ProductFinder/ProductFinder.jsx (hunk 2 above)
Comment (bobmeredith) [reply react-components/src/components/ProductFinder/ProductFinder.jsx:285]: Hmm, don't quite get you there.  You mean you'd rather see a variable set outside the JSX 
`let iconClass = 'fa ' + (selectedProduct ? 'fa-edit' : 'fa-plus')
...
<i className={iconClass}></i>
`

Comment (bobmeredith) [reply react-components/src/components/ProductFinder/ProductFinder.jsx]: All done and linted up

**Pull request merged**